README.md

```diff
@@ -1,10 +1,134 @@
+    - [32. Problem framing](https://open-academy.github.io/machine-learning/machine-learning-productionization/problem-framing.html)
+    - [33. Data engineering](https://open-academy.github.io/machine-learning/machine-learning-productionization/data-engineering.html)
+    - [34. Model training & evaluation](https://open-academy.github.io/machine-learning/machine-learning-productionization/model-training-and-evaluation.html)
+    - [35. Model deployment](https://open-academy.github.io/machine-learning/machine-learning-productionization/model-deployment.html)
+7. **SUPPORTING MATERIALS**
+    - [36. PyTorch](https://open-academy.github.io/machine-learning/supporting-materials/pytorch.html)
+    - [37. Bamboolib](https://open-academy.github.io/machine-learning/supporting-materials/bamboolib.html)
+    - [38. Mito](https://open-academy.github.io/machine-learning/supporting-materials/mito.html)
+    - [39. KNN (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/KNN.html)
+    - [40. Semi-supervised learning (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/semi-supervised-learning.html)
+    - [41. Unbalanced problems](https://open-academy.github.io/machine-learning/supporting-materials/unbalanced-problems.html)
+    - [42. AutoML (TBD)](https://open-academy.github.io/machine-learning/supporting-materials/automl.html)
+8. **OTHERS**
+    - [43. Assignments](https://open-academy.github.io/machine-learning/assignments/introduction.html)
+    - [44. Slides](https://open-academy.github.io/machine-learning/slides/introduction.html)
+
+### Curriculum
+
+We are committed to delivering high-quality and practical courses on schedule every week, featuring instructors from both academia and industry, including technical experts from leading technology companies. Currently, we have completed the section on Data Science and are now in progress with the Machine Learning section. Due to limited capacity and venue availability, the course will be conducted online via [online textbook](https://open-academy.github.io/machine-learning/intro.html) and [VooV (Tencent) meeting](https://www.tencentcloud.com/products/tvm), and at this time, we are only accepting students who are proficient in Chinese (although we plan to address this issue in the future). Please find the class schedule below.
+
+| Time | Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Morning |  |  |  |  |  |  |  |
+| Afternoon |  |  |  |  |  |  |  |
+| Evening |  |  |  |  |  |  |  |
+
+### Community
+
+### Learning Resources
+
+## **How to Contribute**
+
+We welcome contributions from students and educators who are passionate about data science, machine learning, and deep learning. If you'd like to contribute to the project, simply fork the repository and make your changes. You can then submit a pull request to have your changes reviewed and merged into the main repository.

```

Comment:
capitalize the first letter of each word of data science, machine learning, and deep learning

Reply:
Thanks for the comment, I will modify it.